ec2/sc-ec2-linux-jumpcloud.yaml

```diff
@@ -436,15 +435,15 @@ Resources:
       ImageId: !FindInMap [LinuxDistributions, !Ref LinuxDistribution, AMIID]
       InstanceType: !Ref 'EC2InstanceType'
       SubnetId: !If
-        - UsePrivateSubnet
-        - !ImportValue
-          'Fn::Sub': '${AWS::Region}-internalpoolvpc-PrivateSubnet'
+        - UsePublicSubnet
         - !ImportValue
           'Fn::Sub': '${AWS::Region}-internalpoolvpc-PublicSubnet'
+        - !ImportValue
+          'Fn::Sub': '${AWS::Region}-internalpoolvpc-PrivateSubnet'
       SecurityGroupIds: !If
-        - UsePrivateSubnet
+        - UsePublicSubnet
+        - [!Ref PrivateSubnetSecurityGroup, !Ref PublicSubnetSecurityGroup]
         - [!Ref PrivateSubnetSecurityGroup]
```

Comment:
why not just do 
```
- !ImportValue
              'Fn::Sub': '${AWS::Region}-${VpcName}-VpnSecurityGroup
```
like https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/managed-ec2-ubuntu-v1.yaml#L507

Reply:
ohh yeah, reason is to avoid imports in libs